app/src/main/java/com/infomaniak/drive/data/sync/UploadMigration.kt

```diff
@@ -106,6 +106,6 @@ class UploadMigration : RealmMigration {
     }
 
     companion object {
-        const val dbVersion = 6L // Must be bumped when the schema changes
```

Comment:
Unfortunatly in realm java, just bumping the version isn't enough, you have to make a real migration

Reply:
Even if we are just adding something ? 🤔

Reply:
yes